src/app/components/common/node-editor/connection.ts

```diff
@@ -0,0 +1,18 @@
+import { CurveFactory } from 'd3-shape';
+import { Filter, North, Notification, South } from './nodes';
+import { ClassicPreset } from 'rete';
+
+type Node = South | North | Filter | Notification;
```

Comment:
Notification means Rule | Delivery (Notify or Action)?

Reply:
You mean Notification node name represents both the Rule/Delivery action plugin node?

Reply:
![Screenshot 2025-02-12 at 12 52 15](https://github.com/user-attachments/assets/189e93bf-e1a2-488a-9d74-6c08f026bde8)

Reply:
Got it! This is for Notification instances grid page; not the details.

Reply:
There is no details page for Notification.